pkg/logs/config/endpoints.go

```diff
@@ -20,6 +20,12 @@ type Endpoint struct {
 	CompressionLevel        int  `mapstructure:"compression_level" json:"compression_level"`
 	ProxyAddress            string
 	ConnectionResetInterval time.Duration
+
+	BackoffFactor    float64
```

Comment:
Any reason we can't instead just store the backoff policy here instead of passing through all the configs?

Reply:
This was just a style choice. `Endpoint` is also used with `json.Unmarshal` when specifying additional endpoints, it seemed natural to keep it as a flat container of parameters.